stepper.md

```diff
@@ -1,64 +1,214 @@
+6  minPulseWidth                              (0-127)
+7  num steps-per-revolution LSB
+8  num steps-per-revolution MSB
+9  maxSpeed LSB                               
+10 maxSpeed MSB
+11 motorPin1 or directionPin number           (0-127)
+12 motorPin2 or stepPin number                (0-127)
+13 [when interface >= 0x0110000] motorPin3    (0-127)
+14 [when interface >= 0x1000000] motorPin4    (0-127)
+15 [when interface && 0x0000001] enablePin    (0-127)
+16 [optional] pins to invert                  (lower 5 bits = pins:
+                                                XXXXXX1 = invert motorPin1
+                                                XXXXX1X = invert motorPin2
+                                                XXXX1XX = invert motorPin3
+                                                XXX1XXX = invert motorPin4
+                                                XX1XXXX = invert enablePin)
+17 END_SYSEX                                  (0xF7)
+```
+
+**Stepper zero**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop command                            (0x01)
+3  device number                           (0-9)
+14 END_SYSEX                               (0xF7)
+```
+
+**Stepper step**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  step command                            (0x02)
+3  device number                           (0-9)
+4  direction                               (0-1) (0x00 = CW, 0x01 = CCW)
 5  num steps byte1 LSB
 6  num steps byte2
-7  num steps byte3 MSB  (21 bits (2,097,151 steps max))
-8  speed LSB            (steps in 0.01*rad/sec  (2050 = 20.50 rad/sec))
-9  speed MSB
-10 [optional] accel LSB (acceleration in 0.01*rad/sec^2 (1000 = 10.0 rad/sec^2))
+7  num steps byte3 MSB                     (21 bits (2,097,151 steps max))
+8  num speed LSB                           (steps per second)
+9  num speed MSB
+10 [optional] accel LSB                    (acceleration in steps/sec^2)
 11 [optional] accel MSB
-12 [optional] decel LSB (deceleration in 0.01*rad/sec^2)
+12 [optional] decel LSB                    (deceleration in steps/sec^2)
 13 [optional] decel MSB
-14 END_SYSEX            (0xF7)
+14 END_SYSEX                               (0xF7)
+```
+
+**Stepper to**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  to command                              (0x03)
+3  device number                           (0-9)
+4  num target byte1 LSB
+5  num target byte2 MSB
+6  speed LSB                               (steps per second)
+7  speed MSB
+8  [optional] accel LSB                    (acceleration in steps/sec^2)
+9  [optional] accel MSB
+10 [optional] decel LSB                    (deceleration in steps/sec^2)
+11 [optional] decel MSB
+12 END_SYSEX                               (0xF7)
+```
+
+**Stepper enable**
```

Comment:
Do we need separate enable and disable commands? Could this just be enable true | false. Could add one more byte for the state, or OR it with the device number byte.